Made some changes to make this awesome tool a little bit better

fang0654 opened the pull request:
Added command line flags for rest of options
Added additional logic for first and last names
Added CSV writer to write files as UTF-8 instead of UTF-16

There also seemed to be a mix of tabs and spaces for whitespace, I changed it all to just spaces.

fang0654 committed:
Added command line flags for rest of options
Added additional logic for first and last names
Added CSV writer to write files as UTF-8 instead of UTF-16

cloudgate313 commented:
Not getting any information to print to a csv file.  What am I missing?

cloudgate313 commented:
██╗     ██╗███╗   ██╗██╗  ██╗███████╗██████╗ ██╗███╗   ██╗████████╗
██║     ██║████╗  ██║██║ ██╔╝██╔════╝██╔══██╗██║████╗  ██║╚══██╔══╝
██║     ██║██╔██╗ ██║█████╔╝ █████╗  ██║  ██║██║██╔██╗ ██║   ██║   
██║     ██║██║╚██╗██║██╔═██╗ ██╔══╝  ██║  ██║██║██║╚██╗██║   ██║   
███████╗██║██║ ╚████║██║  ██╗███████╗██████╔╝██║██║ ╚████║   ██║   
╚══════╝╚═╝╚═╝  ╚═══╝╚═╝  ╚═╝╚══════╝╚═════╝ ╚═╝╚═╝  ╚═══╝   ╚═╝   
                                                                   
Providing you with Linkedin Intelligence
Author: Vincent Yiu (@vysec, @vysecurity)
Original version by @DisK0nn3cT
[*] Enter search Keywords (use quotes for more percise results)
"manager"
[*] Enter filename for output (exclude file extension)
test
[*] Filter by Company? (Y/N): 
n
[*] Enter e-mail domain suffix (eg. contoso.com): 
gmail.com
[*] Select a prefix for e-mail generation (auto,full,firstlast,firstmlast,flast,first.last,fmlast,lastfirst): 
auto
[*] Automaticly using Hunter IO to determine best Prefix
[!] Rate limited by Hunter IO trial
[!] None
[!] Automatic prefix search failed, please insert a manual choice
[*] Select a prefix for e-mail generation (auto,full,firstlast,firstmlast,flast,first.last,fmlast,lastfirst): 
firstlast

vysecurity commented:
Prints by default to CSV and HTML

On Tue, 3 Jul 2018 at 18:24, Evan Anthony <notifications@github.com> wrote:

> ██╗ ██╗███╗ ██╗██╗ ██╗███████╗██████╗ ██╗███╗ ██╗████████╗
> ██║ ██║████╗ ██║██║ ██╔╝██╔════╝██╔══██╗██║████╗ ██║╚══██╔══╝
> ██║ ██║██╔██╗ ██║█████╔╝ █████╗ ██║ ██║██║██╔██╗ ██║ ██║
> ██║ ██║██║╚██╗██║██╔═██╗ ██╔══╝ ██║ ██║██║██║╚██╗██║ ██║
> ███████╗██║██║ ╚████║██║ ██╗███████╗██████╔╝██║██║ ╚████║ ██║
> ╚══════╝╚═╝╚═╝ ╚═══╝╚═╝ ╚═╝╚══════╝╚═════╝ ╚═╝╚═╝ ╚═══╝ ╚═╝
>
> Providing you with Linkedin Intelligence
> Author: Vincent Yiu (@vysec <https://github.com/vysec>, @vysecurity
> <https://github.com/vysecurity>)
> Original version by @DisK0nn3cT <https://github.com/DisK0nn3cT>
> [
>
> *] Enter search Keywords (use quotes for more percise results) "manager" [*]
> Enter filename for output (exclude file extension)
> test
> [
>
> *] Filter by Company? (Y/N): n [*] Enter e-mail domain suffix (eg.
> contoso.com):
> gmail.com
> [
>
> *] Select a prefix for e-mail generation
> (auto,full,firstlast,firstmlast,flast,first.last,fmlast,lastfirst): auto [*]
> Automaticly using Hunter IO to determine best Prefix
> [!] Rate limited by Hunter IO trial
> [!] None
> [!] Automatic prefix search failed, please insert a manual choice
> [*] Select a prefix for e-mail generation
> (auto,full,firstlast,firstmlast,flast,first.last,fmlast,lastfirst):
> firstlast
>
> —
> You are receiving this because you were mentioned.
>
>
> Reply to this email directly, view it on GitHub
> <https://github.com/mdsecactivebreach/LinkedInt/pull/6#issuecomment-402096714>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ADbf0u5hkngAZQMzfKgWEahdin-K_xrZks5uC0ZFgaJpZM4Q6I2P>
> .
>

Commit:
Made lots of updates

fang0654 committed:
Moved python to top

fang0654 committed:
Added banner into file.

fang0654 committed:
Last change to banner

fang0654 committed:
Removed proxies

absolomb committed:
Update LinkedInt.py

Fixed a small bug when location data wasn't populated the script would error out. Converted to try/except like some of the other data variables. Added in a line to silence the URL warnings and also cleaned up some code that was printing out the session data so it's cleaner for screenshots.

Commit:
Merge pull request #1 from absolomb/patch-1

Update LinkedInt.py

Commit:
Removed http and thready

Commit:
Updated to accept a session cookie instead of l/p
